### ISSUE TITLE
slow interface response with lv2

### ISSUE THREAD

**Issue by farvardin** [jpcima/tunefish#5]
hi,

I've just compiled tunefish using your repo. I got a vst version, and a LV2 version in the src/tunefish4/Builds/LinuxMakefile/build

my OS : linux mint 19.3, with g++ (Ubuntu 7.3.0-27ubuntu1~18.04)

The problem is while the vst version is working rather fine (there are some latency on the interface but it's rather ok), the lv2 version is much slower, about two times slower. When browsing presets or changing parameters it shows much.

**Comment by jpcima** [comment]
Was the attempt made using the same host for vst and lv2?

For short, event processing of VST2 plugin UIs are running on a timer. (I don't know who ever thought this was a good idea)
The fluidity of the UI is submitted to how frequently the host will fire the timer.
For LV2, I recall there exists a property that can set the interval of timer.

**Comment by farvardin** [comment]
Yes, same host, same DAW, same settings. I only set one track to vst, one other to lv2. The lv2 version is significantly slower. Isn't it the same for you?
